### ISSUE TITLE
Nice comments in your code 

### ISSUE THREAD

**Issue by HStewart23** [fac-15/FAC-YAK#26]
The comments in your code make it so readable! Very nice :) 

![](https://media.giphy.com/media/l0MYtRl33WaN4HGBq/giphy.gif)

**Comment by boi87** [comment]
Absolutely. Never found it so easy to scrap code from FACers. Much clear, very easy, WOW.

![](https://media.giphy.com/media/54Vj1kxvgyF4k/giphy.gif)
